### PR TITLE
Publish article to announce the Content style guide release

### PR DESCRIPTION
**Pull request by Julievrz**
Expected publication date: **September 8, 2021** at 3pm please :pray:

### CODE REVIEW

**Review by prestascott — APPROVED** [review]

File: news/_posts/2021-09-08-content-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+![Thumb up](/assets/images/2021/09/build-thumb-up.png)
+
+### Next steps 
+
+From now on, wording reviews will be based on the guidelines in the guide. Any new string added to the project will have to meet the standards. So, if your pull request contains new strings, please use the Content style guide as a reference. 
+
+Regarding existing strings, they'll have to be modified too. That's a lot of work. But together we are stronger! :muscle: If you think that a string could be improved, feel free to [open an issue](https://github.com/PrestaShop/PrestaShop/issues/new/choose) (and why not a pull request?) and make a suggestion based on the guidelines in the guide. 
+
+Also, any suggestions for improvement are welcomed! Nothing is cast in stone. If you think a component is missing, or if you can't find the answer to your question in the guide, you can also open an issue.
```
Comment (matks) [inline news/_posts/2021-09-08-content-style-guide.md:47]: On which github repository should people open issues?

Comment (Julievrz) [reply news/_posts/2021-09-08-content-style-guide.md:47]: @matks At first we wanted to create a specific repo, but Pablo said that it was better to use the main repository for now. If we notice that there are a lot of issues and/or PRs we might consider creating a new repo.

File: news/_posts/2021-09-08-content-style-guide.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+It's a win-win!
+
+![Thumb up](/assets/images/2021/09/build-thumb-up.png)
+
+### Next steps 
+
+From now on, wording reviews will be based on the guidelines in the guide. Any new string added to the project will have to meet the standards. So, if your pull request contains new strings, please use the Content style guide as a reference. 
+
+Regarding existing strings, they'll have to be modified too. That's a lot of work. But together we are stronger! :muscle: If you think that a string could be improved, feel free to [open an issue](https://github.com/PrestaShop/PrestaShop/issues/new/choose) (and why not a pull request?) and make a suggestion based on the guidelines in the guide.
```
[the remark this answers is not in the document]
Comment (Julievrz) [reply news/_posts/2021-09-08-content-style-guide.md:45]: I think @prestascott told Hiba, but I'll make sure to remind the QA team about this. :-)

File: news/_posts/2021-09-08-content-style-guide.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+title: "Content style guide: a reference for better harmonization"
+subtitle: "A reference document for writing content" 
+date: 2021-09-08 15:00:00
+authors: [JulieVarisellaz]
+icon: icon-mobile
+image: /assets/images/2021/09/build-content-style-guide-banner.png
+tags: [wording, ux]
+---
+
+It all started with an observation: contributors brought many new wordings to the PrestaShop open source project but couldn't access clear content guidelines. This led to a lot of inconsistencies. Copy is such an essential element of a good user experience, it was time for a change.
```
[the remark this answers is not in the document]
Comment (Julievrz) [reply news/_posts/2021-09-08-content-style-guide.md]: You're right, "copy" might not be clear enough for everyone. "Wording" seems more commonly used on the project, I'll go with wording. But "copy" basically means written material. :)

File: news/_posts/2021-09-08-content-style-guide.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+![Content style guide banner](/assets/images/2021/09/build-content-style-guide-banner.png)
+
+### A reference document for writing content
+
+When designing a new feature, contributors to the PrestaShop project have to think about the copy: feature labels, help texts, error messages, etc. Because there are many contributors to the project, multiple writers take action on the content by adding new [strings](https://zeroheight.com/80a6c5a61/v/latest/p/36889f-glossary/b/34bc6f). Moreover, the community is very diverse, and every writer has his/her own writing style. To ensure consistency, it was important to provide a document that anyone writing for the project could refer to.
```
[the remark this answers is not in the document]
Comment (Julievrz) [reply news/_posts/2021-09-08-content-style-guide.md]: Done :-)

File: news/_posts/2021-09-08-content-style-guide.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+--- 
+layout: post
+title: "Content style guide: a reference for better harmonization"
+subtitle: "A reference document for writing content" 
+date: 2021-09-08 15:00:00
```
Comment (matks) [inline news/_posts/2021-09-08-content-style-guide.md]:
```suggestion
date: 2021-09-08 16:00:00
```

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Julievrz

**Pull request merged**